### PR TITLE
ID review #24

### PR DESCRIPTION
**Pull request by MaiHameed**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -425,7 +425,8 @@ inventory   inventory-guide.apps.lights.os.fyre.ibm.com         inventory   9085
 Visit the `inventory` application by going to the following URL. Make sure to substitute the appropriate `HOST`:
 `http://[HOST]/inventory/systems`
 
-For example: `\http://inventory-guide.apps.lights.os.fyre.ibm.com/inventory/systems`.
+For example, `inventory-guide.apps.lights.os.fyre.ibm.com` is the HOST in the
+`\http://inventory-guide.apps.lights.os.fyre.ibm.com/inventory/systems` URL.
```
Comment (Charlotte-Holt) [inline README.adoc:429]: @MaiHameed Should "HOST" be in backticks like it is in the previous sentence?

Comment (MaiHameed) [reply README.adoc:429]: Good catch, fixed, thanks :)

**Comment by MaiHameed** [comment]
Hi @Charlotte-Holt, made a few small changes if you could re-review them quickly please? :)

File: README.adoc [hunk 2]
```diff
@@ -6,62 +6,61 @@
 // Contributors:
 //     IBM Corporation
 //
-:projectid: operator-openshift
+:projectid: cloud-openshift-operator
 :page-layout: guide-multipane
 :page-duration: 20 minutes
 :page-releasedate: 2020-07-11
-:page-description: Explore how to deploy microservices using the Open Liberty Operator on Red Hat OpenShift
+:page-description: Explore how to deploy microservices to Red Hat OpenShift by using the Open Liberty Operator
```
Comment (Charlotte-Holt) [inline README.adoc:13]: Could you add a period at the end of this description?

File: README.adoc [hunk 3]
```diff
@@ -6,62 +6,61 @@
...
 :page-tags: ['Kubernetes', 'Docker', 'Cloud'] 
 :page-permalink: /guides/{projectid}
 :page-related-guides: ['cloud-openshift', 'okd']
 :common-includes: https://raw.githubusercontent.com/OpenLiberty/guides-common/master
 :source-highlighter: prettify
-:page-seo-title: Deploying Java microservices using the Open Liberty Operator on Red Hat OpenShift
-:page-seo-description: A getting started tutorial with examples on how to deploy Java microservices using the Kubernetes Operator for Open Liberty on a Red Hat OpenShift cluster.
+:page-seo-title: Deploying Java microservices to Red Hat OpenShift using the Open Liberty Operator
+:page-seo-description: A getting started tutorial with examples on how to deploy Java microservices on a Red Hat OpenShift cluster using the Kubernetes Operator for Open Liberty
```
Comment (Charlotte-Holt) [inline README.adoc:20]: Could you add a period at the end of this too? :)

**Comment by MaiHameed** [comment]
Thanks @Charlotte-Holt!

**Pull request merged**